packages/forms/src/Components/Concerns/CanBeSearchable.php

```diff
@@ -52,6 +54,13 @@ public function searchPrompt(string | Htmlable | Closure | null $message): stati
         return $this;
     }
 
+    public function searchPlaceholder(string | Closure | null $message): static
+    {
+        $this->searchPlaceholder = $message;
+
+        return $this;
+    }
+
```

Comment:
Could you move this above `searchPrompt` to ensure alphabetical sort, @dmandrade?